controllers/apps/componentversion_controller_test.go

```diff
@@ -71,12 +71,14 @@ var _ = Describe("ComponentVersion Controller", func() {
 		cleanEnv()
 	})
 
-	createCompDefinitionObjs := func() []*appsv1alpha1.ComponentDefinition {
+	createCompDefinitionObjs := func(withDefaultServiceVersion bool) []*appsv1alpha1.ComponentDefinition {
```

Comment:
Do not add this flag to the function, just update the object to what you want in BeforeEach of the context.

Reply:
done